### ISSUE TITLE
Tries to display videos in markdown comments

### ISSUE THREAD

**Issue by MV-GH** [dessalines/jerboa#1188]
### Jerboa Version

0.0.44

### Android Version + Phone

n/a

### Describe The Bug

The coil images plugin for markwon tries to display videos. It should just leave the link as is.

https://github.com/dessalines/jerboa/assets/67873169/3ec013fb-0699-482e-b12d-5e289d391073

### To Reproduce

find a comment with mp4 link in comments, watch it display error placeholder as it can not display videos.

**Comment by MV-GH** [comment]
when doing `!{}(vidlink)` it tries to turn into a drawable or something that CoilImage would try to display

**Comment by MV-GH** [comment]
Current behaviour is that it should show the first frame of a video, I am guessing that the end goal that it would open up the video player?

**Comment by dessalines** [comment]
Yes, I'd much prefer it just try to do use whatever program android wants for that video uri.
